lib/fastlane/actions/slack.rb

```diff
@@ -27,11 +27,11 @@ def self.run(options)
 
         notifier = Slack::Notifier.new(options[:slack_url])
 
-        if options[:username].to_s.length > 0
-          notifier.username = options[:username]
-        else
-          notifier.username = 'fastlane'
-        end
+        notifier.username = if options[:username].to_s.length > 0
```

Comment:
I don't like having an `if` next to the `=`. Could you update the `rubocop` rule to not do that any more and maybe even discourage it?